block/modes.go

```diff
@@ -95,6 +93,7 @@ func (m *Manager) runAsProposer(ctx context.Context, eg *errgroup.Group) error {
 		// Check if loops exited due to sequencer rotation signal
 		if errors.Is(err, errRotationRequested) {
 			m.rotate(ctx)
+			m.freezeNode(fmt.Errorf("rotation completed. please restart."))
```

Comment:
we have `panic` in `rotate()` so this freeze won't be called

Reply:
yes, i didnt see it. i added this because i got it stuck here when testing rotation, but maybe was using an older version. removed